windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -337,6 +337,34 @@ procedure Run;
   end;
 end;
 
+function DoCheckTIPInstallStatus(FSilent: Boolean): Boolean;
+begin
+(*
```

Comment:
So, this method was first refactored, then commented-out?

There's one line that seems new and lacks context for reviews...

Reply:
Yeah, I saw the code in two places, so refactored as I went.